packages/core/src/tracing/instrumentation/database/mongodb.js

```diff
@@ -438,11 +449,15 @@ function createWrappedCallback(span, originalCallback) {
 
 function handleCallbackOrPromise(ctx, originalArgs, originalFunction, span) {
   const { originalCallback, callbackIndex } = tracingUtil.findCallback(originalArgs);
+
+  console.log('handleCallbackOrPromise: ', callbackIndex, originalCallback);
+
+  // Probably changes needs to be done here
```

Comment:
Yes. There is a bug in the `findCallback` function.
https://github.com/instana/nodejs/pull/1304